backend/lcfs/web/api/final_supply_equipment/services.py

```diff
@@ -314,3 +320,33 @@ async def get_compliance_report_by_id(self, compliance_report_id: int):
     @service_handler
     async def delete_all(self, compliance_report_id: int):
         return await self.repo.delete_all(compliance_report_id)
+
+    @service_handler
+    async def copy_to_report(
+        self, original_report_id: int, target_report_id: int, organization_id: int
```

Comment:
we want to be copying the previous reports FSE not the original report FSE records. If they update FSE in a supplemental those changes should be carried forward.

Reply:
Correct that is how it works. Is this a question or a suggestion?

Reply:
Should I name this copy_from_id?

Reply:
all good. Just wanted to clarify.